base/deprecated.jl

```diff
@@ -2615,6 +1246,153 @@ end
 @deprecate_moved spones     "SparseArrays" true true
 @deprecate_moved speye      "SparseArrays" true true
 
+# PR #25571: LinearAlgebra to stdlib
+## the LinearAlgebra module itself is deprecated base/sysimg.jl
+
+## functions that where re-exported from Base
```

Comment:
Please feel welcome to ignore the following as entirely unimportant: "that where re-exported" -> "that were re-exported", and likewise in similar comments below :).

Reply:
:sweden: :upside_down_face: